src/overlays/actors/ovl_Bg_Mizu_Movebg/z_bg_mizu_movebg.h

```diff
@@ -4,6 +4,18 @@
 #include "ultra64.h"
 #include "global.h"
 
+typedef enum BgMizuMovebgType {
+    /* 0 */ MIZUMOVEBG_TYPE_FLOAT_PLATFORM_WEST,
+    /* 1 */ MIZUMOVEBG_TYPE_FLOAT_PLATFORM_OUTSIDE_CENTER_PILLAR,
```

Comment:
Are these two swapped?
fast64 import:
![oot's water temple and its bgmizumovebg actors](https://github.com/zeldaret/oot/assets/5306619/f314b741-bb69-41eb-88f5-7c770f166216)
the 0xXXXX value is each actor's params
the left-most nybble (`T` in `0xTXXX`) is the type (`#define MOVEBG_TYPE(params) (((u16)(params) >> 0xC) & 0xF)`)
0 is on the center pillar near the entrance, and 1 is further away to the side (west?)?